ruleset.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0"?>
+<ruleset name="YOUR_PROJECT">
+	<config name="installed_paths" value="../../slevomat/coding-standard"/><!-- relative path from PHPCS source location -->
+
+	<rule ref="PSR2"/>
```

Comment:
Perhaps it would be better for you to publish a new standard to cover "slevomat but for 7.1"? It's a lot to inline here

Reply:
I left rules commented out as reference whilst I was learning how Slevomat rules worked, and I have found rules that oppose each other (this prevents phpcbf fixing files).   Which one to choose then becomes a matter of opinion, or minimal PHP version being aimed at.  A slevomat config for different PHP versions would be ideal, perhaps I should raise that as a feature request :)

Reply:
Yeah having read the Slevomat readme they seem to recommend the approach of inlining individual rules directly so maybe we stick with that for this PR...